packages/ionic/src/commands/deploy/manifest.ts

```diff
@@ -31,10 +32,24 @@ export class DeployManifestCommand extends Command {
       throw new FatalException(`Cannot run ${input('ionic deploy manifest')} outside a project directory.`);
     }
 
-    const buildDir = path.resolve(this.project.directory, 'www'); // TODO: this is hard-coded
+    const integration = await this.getAppIntegration();
+    let buildDir: string;
+    if (integration && ['capacitor', 'cordova'].includes(integration)) {
+      buildDir = await this.project.getDistDir();
+    } else {
+      throw new FatalException(
+        `It looks like your app isn't integrated with Capacitor or Cordova.\n` +
+        `In order to generate a manifestfor the Appflow Deploy plugin, you will need to integrate your app with Capacitor or Cordova. See the docs for setting up native projects:\n\n` +
+        `iOS: ${strong('https://ionicframework.com/docs/building/ios')}\n` +
+        `Android: ${strong('https://ionicframework.com/docs/building/android')}\n`
+      );
+    }
+
     const manifest = await this.getFilesAndSizesAndHashesForGlobPattern(buildDir);
 
-    await writeFile(path.resolve(buildDir, 'pro-manifest.json'), JSON.stringify(manifest, undefined, 2), { encoding: 'utf8' });
+    const manifestPath = path.resolve(buildDir, 'pro-manifest.json');
+    await writeFile(manifestPath, JSON.stringify(manifest, undefined, 2), { encoding: 'utf8' });
+    this.env.log.ok(`Manifest ${input(manifestPath)} correctly generated\n`);
```

Comment:
This will print an absolute path, which aren't used often in CLI output. Instead, use `prettyPath`.

```ts
this.env.log.ok(`Appflow Deploy manifest written to ${input(prettyPath(manifestPath))}!`);
```